sdk/src/auth.rs

```diff
@@ -79,6 +87,7 @@ impl NetworkAuth {
     /// Signs a message to relay a proof.
     pub async fn sign_relay_proof_message(
         &self,
```

Comment:
docs: This is unclear. This should specify that a user is requesting for a hosted relay of a proof.

Reply:
added